OpenShift/helm/nexus-repository-manager/values.yaml

```diff
@@ -9,8 +9,8 @@ deploymentStrategy: {}
   # type: RollingUpdate
 
 nexus:
-  imageName: registry.connect.redhat.com/sonatype/nexus-repository-manager
-  imageTag: 3.21.1-01-ubi-23
+  imageName: sonatype/nexus3
+  imageTag: 3.23.0
```

Comment:
thanks for the upgrade. as in the other values.yaml, could the version be pushed into imageName?

Reply:
🙅 my suggestion is invalid at this point. this should not be combined now.